config/locales/links/es.yml

```diff
@@ -16,12 +16,12 @@ es:
     go_back: Regresa
     help: Ayuda
     new_tab: (abrir nueva pestaña)
-    next: Siguiente
```

Comment:
Not sure if we want to include it in scope here, but I discovered that we're referencing these labels in some places in support articles, so I assume we'd want to keep those in sync with the actual implementation. Might be a question to bring back to the team to discuss how we'd want to address this.

Example: https://github.com/18F/identity-site/blob/e6f6ada/content/_help/manage-your-account/change-your-phone-number._es.md#L17